website/docs/terms/data-warehouse.md

```diff
@@ -1,10 +1,15 @@
 ---
 id: data-warehouse
 title: Data warehouse
+description: How have data warehouses evolved over the last 40 years? Explore the nuanced changes in use case since Inmon first coined the term.
 displayText: data warehouse
 hoverSnippet: A data warehouse is a data management system used for data storage and computing that allows for analytics activities such as transforming and sharing data.
 ---
 
+<head>
+    <title>Data warehouses in the post-modern data stack - dbt Labs</title>
```

Comment:
```suggestion
    <title>Data warehouses in the modern data stack - dbt Labs</title>
```

Reply:
Any reason why you don't agree with phrasing here? I'm trying to give the impression that this is a more updated definition then everything else on page one

Reply:
not quite sure what a post-modern data stack is...seems a little clickbait-y